src/Verify.Terminal/Commands/ReviewCommand.cs

```diff
@@ -81,41 +86,38 @@ public sealed class Settings : CommandSettings
     }
 
     private static SnapshotAction ShowPrompt()
```

Comment:
There is no longer a loop until the user submits a valid command. We need to add that.